src/bootstrap/setup.rs

```diff
@@ -19,6 +19,33 @@ pub enum Profile {
     User,
 }
 
+static VSCODE_SETTINGS: &str = r#"{
```

Comment:
nit: can you make this a standalone file in src/etc so it's easy to modify and copy without having to use x.py setup?